CHANGELOG.md

```diff
@@ -16,4 +16,14 @@ All notable changes to this project will be documented in this file.
  * Added postgresql DB
  * TLS configuration for Authentication and Postgres
  * Removed Hashicorp Vault dependencies
- 
+
+## v1.1.4 (2021-12-08)
```

Comment:
I'm not sure to understand how versioning is done in this repo

Reply:
What do you mean ?